packages/typespec-ts/test/modularUnit/scenarios/operations/cookieParam/ignoreCookieParam.md

```diff
@@ -23,25 +23,29 @@ mustEmptyDiagnostic: false
 Should normal path parameter:
 
 ```ts operations
-import { TestingContext as Client } from "./index.js";
+import { TestingContext as Client } from "../index.js";
 import {
   StreamableMethod,
   PathUncheckedResponse,
   createRestError,
-  operationOptionsToRequestParameters
+  operationOptionsToRequestParameters,
 } from "@azure-rest/core-client";
 
 export function _testSend(
   context: Client,
-  options: TestOptionalParams = { requestOptions: {} }
+  token: string,
```

Comment:
cookie param is not being ignored?

Reply:
resolved by this temporary fix https://github.com/Azure/autorest.typescript/pull/2943#discussion_r1881924203